cxx/include/mspass/utility/MetadataDefinitions.h

```diff
@@ -28,16 +28,17 @@ data file.
 class MetadataDefinitions
 {
 public:
-  /*! Default constructor - defaults to mspass namespace. */
-  MetadataDefinitions();
+  /*Comment out default constructor for now - needs to default to mspass 
+   * and not sure how to do that.
+  MetadataDefinitions();*/
```

Comment:
I guess this is already done by the the default of following constructor `MetadataDefinitions(const std::string mdname=std::string("mspass"))`, right?

Reply:
Yes, this is the issue I wasn't sure how to resolve so I pushed it up for now.   Let me experiment with this a bit today.   I think there may be a simple solution, but I need to test my idea.   Glad you caught this as I had forgotten this little titbit I had abandoned when trying to catch bigger fish.